book/src/component-model/overview.md

```diff
@@ -0,0 +1,16 @@
+## Component Model
+
+This section explains the Harbor **component model**. The DoD [Architecture Framework](https://dodcio.defense.gov/Portals/0/Documents/DODAF/Vol_1_Sect_7-2-1_Component_Models.pdf)
```

Comment:
The document also says that a Component Model is described using three views:
1. Component Relationship Diagram
2. Component Description
3. Component Interaction Diagram 

Are these to be developed?  Are they already here somewhere?

Reply:
Couldn't fit that in the first pass, but it would be a great goal to flesh this out with those additional sections.